tests/lib/rules/no-global-assertions.js

```diff
@@ -9,17 +9,14 @@
 //------------------------------------------------------------------------------
 
 const rule = require("../../../lib/rules/no-global-assertions"),
-    RuleTester = require("eslint").RuleTester;
+    RuleTester = require("eslint").RuleTester,
+    testUtils = require("../../testUtils");
 
 
 //------------------------------------------------------------------------------
 // Helpers
 //------------------------------------------------------------------------------
 
-function wrap(assertionCode, testName = "Name") {
```

Comment:
Some versions of wrap contain a testName argument, but it was never used. If it's needed in the future it can be added back.